js/marquee.js

```diff
@@ -1,12 +1,18 @@
 //marquee.js
 
-	var assocWrap = document.getElementById('assoc_slide_wrap');
-	var pos = -210;
-	var moveMarquee = setInterval ("marquee()", 4000);
+	// Here's another way to declare multiple vars:
+	var assocWrap = document.getElementById('assoc_slide_wrap'),
+			pos = -210,
+			moveMarquee = setInterval ("marquee()", 4000);
 
 	function marquee() {
 		if (pos >= -3390){
-			position = pos + 'px';
```

Comment:
This will technically work, but it's good to get in the habit of always initializing variables with the `var` keyword.